java/src/main/java/ai/rapids/cudf/HostColumnVector.java

```diff
@@ -765,21 +799,39 @@ private void initAndResizeOffsetBuffer(int currentIndex) {
       }
     }
 
-    private void initValidBuffer() {
+    private void initAndResizeValidBuffer() {
       if (this.valid == null) {
         allocateBitmaskAndSetDefaultValues();
```

Comment:
This sets the value based off of the estimated row count only.  What happens if I insert a null after I grew the size beyond the initial estimate?  I would prefer to have us delete this method totally and just have a separate check to decide if we copy or not (don't copy if the data buffer didn't exist before)